src/main/java/be/seeseemelk/mockbukkit/entity/EntityEquipmentMock.java

```diff
@@ -231,8 +240,12 @@ public void setBoots(ItemStack boots, boolean silent)
 	@Override
 	public void setArmorContents(ItemStack[] items)
```

Comment:
No annotation here?

Reply:
We don't know what to do if we give a different array size. Should we add the first 4 items ? Should we do nothing ? Should we throw an Exception ?